yt_dlp/extractor/tver.py

```diff
@@ -202,6 +206,8 @@ def _real_extract(self, url):
                 'description': ('description', {str}),
                 'release_timestamp': ('viewStatus', 'startAt', {int_or_none}),
                 'episode_number': ('no', {int_or_none}),
+                'series_id': ('seriesID', {str_or_none}),
+                'season_id': ('seasonID', {str_or_none}),
```

Comment:
just checking the type should be enough
```suggestion
                'series_id': ('seriesID', {str}),
                'season_id': ('seasonID', {str}),
```